e2e/pages/app/app.module.ts

```diff
@@ -5,219 +5,232 @@ import { RouterModule, Routes } from '@angular/router';
 import { AppComponent } from './app.component';
 
 const routes: Routes = [
+    {
+        path: 'badge',
+        loadChildren: () => import('./badge/badge.module').then((m) => m.BadgeTestPageModule),
```

Comment:
As noted above, I'd appreciate it if you would revert the unnecessary changes in this file.

(We do actually prefer omitting the brackets from single-argument arrow functions in our code.)